### DISCUSSION TITLE
Pooling configuration

### DISCUSSION THREAD

**Discussion by FlagOut** [Chia-Network/chia-blockchain#7228] [Support]
Pool not showing up across multiple computers using same keys. Only showing on the pc that was created. I tried to delete keys on one of the others pc's and put it againg, but still not showing the pool nft. How i load the pool nft on all the pc's with the same keys? Or it's a bug?

**Comment by joelrb** [comment]
Wait for it to fully sync. It will be there. NFT data is in the blockchain so Chia app needs to be in sync.

**Reply by FlagOut** [reply]
Thanks for the reply. I found the problem: "IMPORTANT: If you are using the same key (24 words) on more than one computer, you should update both computers to 1.2+ before creating your plot NFT. If you have already created the plot NFT and the second computer is running a version older than 1.2, it will not be able to see the plot NFT. This can be fixed by shutting down the application on the second computer, and deleting the ~/.chia/wallet/db folder and all files inside of it."

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
